api/managementpb/actions.proto

```diff
@@ -20,86 +20,132 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 };
 option go_package = "managementpb";
 
-import "agentpb/agent.proto";
 import "github.com/mwitkow/go-proto-validators/validator.proto";
 import "google/api/annotations.proto";
 import "protoc-gen-swagger/options/annotations.proto";
 
-// RunActionRequest describes request to run an action.
-message RunActionRequest {
```

Comment:
"Run" is the whole interval from start to finish. We do not wait for the action to finish, so we call this API "start".